src/libs/oci/src/lib.rs

```diff
@@ -22,6 +22,21 @@ pub use serialize::{to_string, to_writer, Error, Result};
 
 pub const OCI_SPEC_CONFIG_FILE_NAME: &str = "config.json";
 
+pub fn get_env_map(env: &Vec<String>) -> std::collections::HashMap<String, String> {
```

Comment:
Added to oci crate since it's already used by the agent and genpolicy. But there may be a better place for this

Reply:
Use new policy crate recently added upstream

Reply:
I'd vote for temporarily using this crate in our fork. Then once I downstream the new policy crate from upstream, moving it there

Reply:
I would prefer to duplicate the code in both callers until we'll get the proper common lib.